### PR TITLE
Add config.yml for behaviorbot/welcome (#55)

### PR DESCRIPTION
**Pull request by zeke**
A re-do of https://github.com/github/docs/pull/55 which got lost in a recent force-push.

### CODE REVIEW

**Review by heiskr — APPROVED** [review]
Sure

**Review by kkash18 — COMMENTED** [review]
#

**Review by kkash18 — COMMENTED** [review]
Very well done #96

**Review by OCTOIAF — COMMENTED** [review]
re-add-welcome-bothttps://github.com/github/docs/blob/2c381349fc1a71a1eec9e0ee7c84c8a367467855/.github/CODEOWNERS#L10

File: .github/config.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Configuration for welcome - https://github.com/behaviorbot/welcome
+
+# Configuration for new-issue-welcome - https://github.com/behaviorbot/new-issue-welcome
+# Comment to be posted to on first time issues
+newIssueWelcomeComment: >
+  Thanks for opening this issue. A GitHub docs team member should be by to give feedback soon. In the meantime, please check out the [contributing guidelines](https://github.com/github/docs/blob/main/CONTRIBUTING.md).
+
+# Configuration for new-pr-welcome - https://github.com/behaviorbot/new-pr-welcome
```
Comment (kas512) [inline .github/config.yml:8]: Teşekkürler

**Review by Girlsavenue2 — COMMENTED** [review]
#22627

**Review by nitindudeja2107 — COMMENTED** [review]
Please approve

**Pull request merged**